common/src/main/java/fudge/notenoughcrashes/stacktrace/ModIdentifier.java

```diff
@@ -190,6 +195,24 @@ else if (NecPlatform.instance().isDevelopmentEnvironment()) {
         }
     }
 
+    /**
+     * Mixin is rather protective of its internal mixin structures.
+     * <p>
+     * Mixin has some useful classes:
+     * <ul>
+     *     <li>{@link ClassInfo} - Represents a class</li>
+     *     <li>{@link IMixinInfo} - Represents a mixin</li>
+     * </ul>
+     * The issue arises when we have a {@link ClassInfo} of a mixin and want to get its {@link IMixinInfo}.
+     * The ClassInfo class has a {@link ClassInfo#mixin mixin} field, used to link an IMixinInfo to a mixin's ClassInfo.
+     * However, this field is private and there is no getter for it. We would have to use reflection to get it.
+     * <p>
+     * Another option would be to get the ClassInfo of the mixin's target class to get mixins into that class.
+     * This has a problem, though, when Mixin does not consider the mixin to have been applied to the target class.
+     * Since it only has a getter for applied mixins and not all mixins, reflection is the only option.
```

Comment:
If a mixin has not been applied to a class, how can it be a potential cause for a crash?

Reply:
Mixin (for some reason) thinks that the test mod's force crash mixin isn't applied

Reply:
Mixin claims that a given mixin hasn't been applied even when it has been